background.ts

```diff
@@ -1243,7 +1243,7 @@ function newK8sManager() {
   const arch = process.arch === 'arm64' ? 'aarch64' : 'x86_64';
   const mgr = K8sFactory(arch, dockerDirManager);
 
-  mgr.on('state-changed', (state: K8s.State) => {
+  mgr.on('state-changed', async(state: K8s.State) => {
```

Comment:
Now that the callback is `async` we should add a `try/catch (ex) { console.error(ex)}` so that we can see that an error happened.  Otherwise the exception just gets dropped on the floor (if, for example, `Steve.getInstance().start()` fails, or some line after that).